packages/react-dom/src/events/EventPluginRegistry.js

```diff
@@ -40,15 +35,7 @@ function publishEventForPlugin(
   eventTypes: EventTypes,
   eventName: string,
 ): boolean {
-  invariant(
```

Comment:
While this invariant is useful to us in theory we have a similar one that would catch the same issue below anyway. We should also probably make that one a DEV-only warning.